stratus/settings.py

```diff
@@ -37,6 +37,8 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'django.contrib.flatpages',
+    'django.contrib.sites',
```

Comment:
Alphabetical please.

Reply:
thats what I thought but sessions messages and staticfiles are not alphabetical....

Reply:
Yeah, nvm, the order can matter.